modules/countdown.js

```diff
@@ -15,14 +26,10 @@ module.exports = {
       }
     }
   },
-  onload: function (bot) {
-    function autoCount (bot, lastTick) {
-      const thisTick = moment('2019-03-29T23:00:00Z').countdown().toString().split(/, | and /)[0]
-      if (thisTick !== lastTick) {
-        bot.broadcast('Article 50 expires in ' + lastTick)
-      }
-      setTimeout(autoCount, 1000, bot, thisTick)
+  events: {
+    join: function (bot, channel, nick, self) {
+      if (timeout !== null || !self) return
```

Comment:
I think this might cause two countdowns to occur if the bot joins multiple channels.
Perhaps instead of using `bot.broadcast()` with one timer, it should `bot.notice(channel)` with multiple timers?